demo-snippet.html

```diff
@@ -157,20 +153,23 @@
 
         // Boolean properties are displayed as checked="", so remove the ="" bit.
         snippet = snippet.replace(/=""/g, '');
-        this._markdown = '```\n' + snippet + '\n' + '```';
+        this._markdown = '```\n' + snippet + '\n' +
```

Comment:
p sure this line was < 80 chars, why did it get split up?

Reply:
this is a fabulous question. I'm going to have to put off debugging until after the I/O rush though.

I ran this through `clang-format` and it did not newline it. This might be an issue with my line parsing in webmat.

Reply:
why is this not one string? 
```js
'\n```'
```

instead of
```js
'\n' + '```';
```

Reply:
Also I can confirm that this is caused by clang-format newlining strings that are just added to each other. I assume this is for breaking up long strings